src/site/content/en/spaces/space-x1-test/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Title Goes Here Title Goes Here Title Goes Here
+description: We’re learning all about Vertex AI this week as Carter Morgan and Jay Jenkins Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+tags: twitter-space
+hosts:
+  - name: Name 1 here
+    twitter: devnook
+    image: https://pbs.twimg.com/profile_images/519869363344515072/s8vGg8fE_400x400.jpeg
+    host: true
```

Comment:
See previously comment—I think this would be clearer if it was `isPrimary` instead of `host.`